test/controllers/api/v2/repository_sets_controller_test.rb

```diff
@@ -103,6 +103,36 @@ def test_available_repositories
       assert_response :success
     end
 
+    def test_available_repo_sort
+      task = assert_sync_task ::Actions::Katello::RepositorySet::ScanCdn
+
+      repo_set = lambda { |version: "", arch: "", path: ""|
```

Comment:
What made you choose to use a lambda here?

Reply:
I got a rubocop offense telling me not to next methods. I asked in the foreman-dev and was told that it was standard to go with a lambda or proc.

Reply:
Cool, I was just curious as to how you arrived there 👍